jib-core/src/main/java/com/google/cloud/tools/jib/docker/json/DockerLoadManifestTemplate.java

```diff
@@ -39,14 +39,18 @@
  * @see <a href="https://github.com/moby/moby/blob/master/image/tarexport/load.go">Docker load
  *     source</a>
  */
-class DockerLoadManifestTemplate implements JsonTemplate {
+public class DockerLoadManifestTemplate implements JsonTemplate {
 
   private final String config = "config.json";
-  private final List<String> repoTags = new ArrayList<>(1);
+  private final List<String> repoTags = new ArrayList<>();
   private final List<String> layers = new ArrayList<>();
 
   public void setRepoTags(String repoTags) {
-    this.repoTags.add(repoTags);
+    if (this.repoTags.isEmpty()) {
```

Comment:
This is okay, though an alternative could be to just set it to an `ImmutableList.of(repoTags)` or have `repoTags` be initialized as a singleton list of `null` to avoid this logic here.